apps/web/src/components/settings/AccountSettings.tsx

```diff
@@ -5,33 +5,55 @@ import { Button } from "@/components/shadcn/ui/button";
 import { Label } from "@/components/shadcn/ui/label";
 import { toast } from "sonner";
 import { useState } from "react";
-import { users } from "db/schema";
-import { z } from "zod";
 import { useAction } from "next-safe-action/hook";
 import { modifyAccountSettings } from "@/actions/user-profile-mod";
+import { Checkbox } from "@/components/shadcn/ui/checkbox";
+import c from "config";
+import { Loader2 } from "lucide-react";
 
 interface UserProps {
```

Comment:
nit: There is an easier way to do this with just importing the table itself and using the `inferSelect` or using zod through the `z.infer` I believe. This serves to benefit us in the fact that it also becomes dynamic so this interface will never need to be updated.

Reply:
Adding on to this, and non-blocking for now, but super common types like this should live in a `types.ts` inside of our db package. For now, keep it in this file though. Just something worth noting.